### PR TITLE
Add barcode printing on badges

### PR DESCRIPTION
**Pull request by maulkin**
This adds the pretty barcode to the badges so we can scan them later.

### CODE REVIEW

**Comment by maulkin** [comment]
Example badge output: [badges-20161022-134806-8a9fd.pdf](https://github.com/CambridgeBeerFestival/Staffunteering/files/546372/badges-20161022-134806-8a9fd.pdf)

**Comment by maulkin** [comment]
Example badge output: 
[badges-20161022-134806-8a9fd (1).pdf](https://github.com/CambridgeBeerFestival/Staffunteering/files/546373/badges-20161022-134806-8a9fd.1.pdf)

**Comment by maulkin** [comment]
Now also adding car pass printing for #5, sample output: 
[wibble.pdf](https://github.com/CambridgeBeerFestival/Staffunteering/files/546444/wibble.pdf)

File: tools/badgegen.py [hunk 1]
```diff
@@ -144,6 +145,9 @@ def Render(self, data, colour=None):
         if data['id']:
             self.canvas.setFont("Courier", 8)
             self.canvas.drawRightString(left + width - 1*mm, bottom+3*mm, data['id'])
+            # Barcode
+            code = code128.Code128(data['id'])
+            code.drawOn(self.canvas, left - 3*mm, bottom+2*mm)
```
Comment (bertkenward) [inline tools/badgegen.py]: Isn't this off the edge?

Comment (maulkin) [inline tools/badgegen.py]: Seems ok: 

[barcodecheck.pdf](https://github.com/CambridgeBeerFestival/Staffunteering/files/547585/barcodecheck.pdf)

Fixes for long job name (#7) to be done.

**Comment by maulkin** [comment]
Now dynamically sets job title (Closes: #7). And I really should work on branches I guess.

**Pull request merged**